### DISCUSSION TITLE
Version 5.1.9.Final is missing in maven

### DISCUSSION THREAD

**Discussion by eitan-rosenberg** [belaban/JGroups#547] [Q&A] [answered]
Just ran versions:display-dependency-updates in maven but 5.1.9.Final is missing.

![image](https://user-images.githubusercontent.com/48506054/129934645-6e0ce0aa-24b7-4747-8cbf-e2984d974d6b.png)

**Comment by belaban** [comment]
I uploaded 5.1.9 to [1]. Sometimes it takes an hour to replicate to maven central...

[1] https://repository.jboss.org/org/jgroups/jgroups/5.1.9.Final/

**Comment by rhusar** [comment]
Hm, still nothing on central: https://search.maven.org/search?q=g:org.jgroups%20AND%20a:jgroups

**Comment by pruivo** [comment, marked as answer]
I can see 5.1.9.Final here: https://repo1.maven.org/maven2/org/jgroups/jgroups/5.1.9.Final/

**Reply by rhusar** [reply]
Looks like its also here as well https://repo.maven.apache.org/maven2/org/jgroups/jgroups/5.1.9.Final/

**Reply by rhusar** [reply]
OK, looks like CI builds are able to fetch this now as well (e.g. here https://github.com/jgroups-extras/jgroups-azure/pull/97 that previously failed)

**Comment by eitan-rosenberg** [comment]
Hi,

Looks ok now.

Thanks.

![image](https://user-images.githubusercontent.com/48506054/130047816-6cf81777-fbc6-470a-a9ad-68a7561137ba.png)

**Reply by rhusar** [reply]
Now, I assume you have a button somewhere to resolve this question..
